etc/config/go.amazon.properties

```diff
@@ -110,11 +110,14 @@ compiler.gl1190.goroot=/opt/compiler-explorer/golang-1.19/go
 compiler.gl1200.exe=/opt/compiler-explorer/golang-1.20/go/bin/go
 compiler.gl1200.semver=1.20
 compiler.gl1200.goroot=/opt/compiler-explorer/golang-1.20/go
+compiler.gl1210.exe=/opt/compiler-explorer/golang-1.21/go/bin/go
```

Comment:
This path is now out of date: the changes in infra means these paths are all now: `/opt/compiler-explorer/golang-1.21.0`

Reply:
Done!